server/protocols/jmap/src/main/java/org/apache/james/jmap/model/mailbox/Role.java

```diff
@@ -41,12 +41,12 @@
     public static final Role OUTBOX = new Role("outbox", DefaultMailboxes.OUTBOX);
     public static final Role SENT = new Role("sent", DefaultMailboxes.SENT);
     public static final Role TRASH = new Role("trash", DefaultMailboxes.TRASH);
-    public static final Role ARCHIVE = new Role("archive");
-    public static final Role SPAM = new Role("spam");
-    public static final Role TEMPLATES = new Role("templates");
+    public static final Role ARCHIVE = new Role("archive", DefaultMailboxes.ARCHIVE);
+    public static final Role SPAM = new Role("spam", DefaultMailboxes.SPAM);
+    public static final Role TEMPLATES = new Role("templates", DefaultMailboxes.TEMPLATES);
     
     private static final Map<String, Role> ROLES = 
-            ImmutableList.<Role>of(INBOX, ARCHIVE, DRAFTS, OUTBOX, SENT, TRASH, SPAM, TEMPLATES)
+            ImmutableList.<Role>of(INBOX, DRAFTS, OUTBOX, SENT, TRASH, ARCHIVE, SPAM, TEMPLATES)
```

Comment:
why did you change the ordering ? Why don't you sort alphabetically to make it easy for a human to search in it ?

Reply:
To match the order of the constant just the lines above.